src/coreclr/tests/issues.targets

```diff
@@ -367,9 +337,6 @@
         <ExcludeList Include="$(XunitTestBinBase)/GC/LargeMemory/API/gc/suppressfinalize/*">
             <Issue>3392, test is useful to have because it can be run manually when making changes to the GC that can have effects in OOM scenarios, but not appropriate to run on our current test infrastructure.</Issue>
         </ExcludeList>
-        <ExcludeList Include="$(XunitTestBinBase)/JIT/opt/Tailcall/TailcallVerifyWithPrefix/*">
-            <Issue>Depends on implicit tailcalls to be performed</Issue>
-        </ExcludeList>
```

Comment:
TailCallVerifyWithPrefix has tests that verify fast tail calls. Since we don't support fast tail calls on x86 we can't remove this exclusion. For example, this test fails on x86:
```
Executing Condition10.Test2 - Caller: Arguments: None - ReturnType: void; Callee<string>: Arguments: None - ReturnType: void
      FAILED: Found the word 'Caller' in the stacktrace.
      ------------------------------------------------
      
         at TailcallVerify.Condition10.Callee2[T]()
         at TailcallVerify.Condition10.Caller2()
         at TailcallVerify.Condition10.Test2()
      
      Execution finished - Test FAILED
```

I'm confused because I think CI was green for your PR at some point but I don't understand how this test could've passed.

Reply:
It looks like I merged this badly when I squashed it originally and migrated it to runtime. In https://github.com/dotnet/coreclr/pull/26418 this change is not there. I will readd this exclusion.